.vscode/launch.json

```diff
@@ -25,7 +25,10 @@
       "internalConsoleOptions": "openOnSessionStart",
       "autoAttachChildProcesses": true,
       "console": "internalConsole",
-      "outputCapture": "std"
+	  "outputCapture": "std",
+	  "env": {
+		  // "MRE_LOGGING": "network"

```

Comment:
>// "MRE_LOGGING": "network" [](start = 4, length = 27)

Comments are not valid in JSON, but maybe it's fine in vscode.

Reply:
VS Code didn't complain. I find myself putting this in there fairly regularly, it'll save us time next time to have this here.

---
In reply to: [335211607](https://github.com/microsoft/mixed-reality-extension-sdk/pull/455#discussion_r335211607) [](ancestors = 335211607)

Reply:
👍